app/src/main/java/com/immichframe/immichframe/MainActivity.kt

```diff
@@ -856,4 +856,34 @@ class MainActivity : AppCompatActivity() {
         rcpServer.stop()
         handler.removeCallbacksAndMessages(null)
     }
+
+    private fun loadWebViewWithRetry(url: String, attempt: Int = 1, maxAttempts: Int = 36) {
```

Comment:
This method is duplicated across the two activities for consistency, since they already have mostly identical methods, but I'm happy to move to Helpers.kt or elsewhere if you prefer?